File: LeetCode-Programs/Remove Duplicates from Sorted Array II.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        '''for i in range(1, len(nums)):
            if nums[i-1]==nums[i+1]:
                nums.remove(nums[i])
                i=i-1
        return len(nums)'''
        s=set(nums)
        for i in s:
            while nums.count(i) > 2:
                nums.remove(i)
        return len(nums)

File: LeetCode-Programs/Remove Duplicates from Sorted Array II.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        if(nums.size()==0)
            return 0;
        for(int i=1;i<nums.size()-1;i++)
        {
            if(nums[i-1]==nums[i+1])
            {
                nums.erase(nums.begin()+i);
                i--;
            }
        }
        return nums.size();
    }
};
